src/download-plugin-manager.tsx

```diff
@@ -46,7 +45,7 @@ class DownloadPluginManager extends KalturaPlayer.core.FakeEventTarget {
       title,
       text,
       icon: <Icon id={`download-started-toast-icon`} path={DOWNLOAD} viewBox={`0 0 32 32`} />,
-      severity: ToastSeverity.Success
+      severity: 'Success'
```

Comment:
after you import `ToastSeverity` from ui-managers you can use the enum again here and for 'Error'

Reply:
cant export enums from *.d.ts files